Classes/FlowEntitiesTrait.php

```diff
@@ -5,14 +5,15 @@
 namespace Neos\Behat;
 
 use Behat\Hook\BeforeScenario;
-use Doctrine\DBAL\DBALException;
+use Doctrine\DBAL\Exception as DoctrineException;
 use Doctrine\DBAL\Schema\Schema;
 use Doctrine\ORM\EntityManagerInterface;
-use Neos\Flow\Persistence\Doctrine\Service;
+use Neos\Flow\Configuration\ConfigurationManager;
+use Neos\Flow\Persistence\Doctrine\Service as FlowDoctrineService;
 use Neos\Flow\Persistence\PersistenceManagerInterface;
 
 /**
- * Tag your test with @ flowEntities to enable support for flow entities
+ * Tag your test with [at]flowEntities to enable support for flow entities
```

Comment:
Why did you replace the `@`?

Reply:
I saw that pattern at another place in flow ;)